roles/filetree_create/templates/current_credentials.j2

```diff
@@ -3,19 +3,24 @@
 controller_credentials:
 {% endif %}
   - name: "{{ current_credentials_asset_value.name }}"
-    description: "{{ current_credentials_asset_value.description }}"
+    description: "{{ template_overrides_global.credential.description | default(current_credentials_asset_value.description) }}"
```

Comment:
It would make sense to let only from `template_overrides_resources` instead of only from `template_overrides_global`...

Reply:
This should be resolved.

Reply:
Sometimes it would be good to set global description, because maybe someone wants to use the description as some kind of indicator.

Reply:
> Sometimes it would be good to set global description, because maybe someone wants to use the description as some kind of indicator.

Ok, but it would make sense to also allow to set the description from `template_overrides_resources` for completeness.